newm-server/src/integTest/kotlin/io/newm/server/SongsTest.kt

```diff
@@ -0,0 +1,175 @@
+package io.newm.server
+
+import com.google.common.truth.Truth.assertThat
+import io.ktor.client.call.body
+import io.ktor.client.request.bearerAuth
+import io.ktor.client.request.forms.formData
+import io.ktor.client.request.forms.submitFormWithBinaryData
+import io.ktor.client.request.get
+import io.ktor.client.request.headers
+import io.ktor.client.request.post
+import io.ktor.client.request.setBody
+import io.ktor.client.request.url
+import io.ktor.client.statement.HttpResponse
+import io.ktor.http.ContentType
+import io.ktor.http.Headers
+import io.ktor.http.HttpHeaders
+import io.ktor.http.HttpStatusCode
+import io.ktor.http.append
+import io.ktor.http.contentType
+import io.ktor.utils.io.core.buildPacket
+import io.ktor.utils.io.core.writeFully
+import io.newm.server.features.model.CountResponse
+import io.newm.server.features.song.model.AudioUploadRequest
+import io.newm.server.features.song.model.AudioUploadResponse
+import io.newm.server.features.song.model.Song
+import io.newm.server.features.song.model.SongIdBody
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withTimeout
+import org.junit.jupiter.api.Test
+import java.io.File
+import java.time.Duration
+import java.util.concurrent.TimeoutException
+
+class SongsTest {
+    @Test
+    fun `Add a new song`(): Unit = runBlocking {
+        val createSongResponse = createSong(
+            Song(
+                title = "[TEST] ${String.randomString(8)}",
+                genres = listOf("Synthwave", "Synthpop")
+            )
+        )
+        assertThat(createSongResponse.status).isEqualTo(HttpStatusCode.OK)
+        val resp = createSongResponse.body<SongIdBody>()
+        assertThat(resp.songId)
```

Comment:
What are you asserting here?

Reply:
probably `.isNotNull()`

Reply:
Could checked that is parsable as a UUID:
```
assertThat(resp.songId).isNotNull()
```